src/main/webapp/homepage.html

```diff
@@ -22,10 +22,10 @@
   </head>
   <body>
     <div id="content">
-        <div id="search-form">
+        <form id="search-form-homepage" onsubmit="return redirectToResults();">
```

Comment:
For the future, it might be better in terms of user experience to be able to trigger redirectToResults() by just pressing enter on the keyboard. This might help: https://www.w3schools.com/howto/howto_js_trigger_button_enter.asp

Reply:
Because it's a form, it automatically has the functionality.